X/xorriso/build_tarballs.jl

```diff
@@ -0,0 +1,62 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "xorriso"
+version = v"1.5.5"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/gnu-mirror-unofficial/xorriso/raw/refs/heads/master/xorriso-$(version).tar.gz", "89a78b902ded443c3e4b31b3ba586ccbf06a447836d37f1082dfb1e429952217")
+]
```

Comment:
It seems there is a relation between GNU xorriso and libburnia xorriso: https://www.gnu.org/software/xorriso/

Would it be worthwhile to consider packaging this as `GNUXorriso`?

Reply:
I updated the official link. I missed it as I was just playing around to see if there were chances of cross-compiling the project before getting in the weeds of the custom build system that cdrtools had come up with.